section-09-differential-testing/model-package/tests/test_prediction.py

```diff
@@ -0,0 +1,22 @@
+import math
+
+import numpy as np
+
+from regression_model.predict import make_prediction
+
+
+def test_make_prediction(sample_input_data):
+    # Given
+    expected_first_prediction_value = 113422
```

Comment:
We should check more than 1 prediction. See one of my first comments.  Also, it would be nice to check model performance as well.

Reply:
I'd leave these, I won't be touching on them at all - they're copies from previous sections